src/Crypto/Secp256k1.hs

```diff
@@ -89,7 +92,7 @@ import Crypto.Secp256k1.Internal
 import Crypto.Secp256k1.Prim (flagsEcUncompressed)
 import Crypto.Secp256k1.Prim qualified as Prim
 import Data.ByteArray.Encoding qualified as BA
-import Data.ByteArray.Sized
+import Data.ByteArray.Sized hiding (take)
```

Comment:
rather than hiding `take` here you should qualify this import as well as the conflicting one. I'd like to move towards a practice of using qualified imports more aggressively.